prime-angular-frontend/src/app/shared/components/forms/fields/doing-business-as-form-field/doing-business-as-form-field.component.html

```diff
@@ -1,13 +1,13 @@
-<!--<mat-form-field class="w-100">-->
-<!--  <input matInput-->
-<!--         [placeholder]="label"-->
-<!--         [formControl]="doingBusinessAs"-->
-<!--         [matAutocomplete]="auto">-->
-<!--  <mat-autocomplete #auto="matAutocomplete">-->
-<!--    <mat-option *ngFor="let doingBusinessAsName of doingBusinessAsNames"-->
-<!--                [value]="doingBusinessAsName">-->
-<!--      {{ doingBusinessAsName }}-->
-<!--    </mat-option>-->
-<!--  </mat-autocomplete>-->
-<!--  <mat-error>Required</mat-error>-->
-<!--</mat-form-field>-->
+<mat-form-field class="w-100">
+  <input matInput
+         [placeholder]="label"
+         [formControl]="doingBusinessAs"
+         [matAutocomplete]="auto">
+  <mat-autocomplete #auto="matAutocomplete">
+    <mat-option *ngFor="let doingBusinessAsName of doingBusinessAsNames"
+                [value]="doingBusinessAsName">
+      {{ doingBusinessAsName }}
+    </mat-option>
+  </mat-autocomplete>
+  <mat-error>Required</mat-error>
+</mat-form-field>
```

Comment:
there's a failing test on this file, checking commit history made me run the tests to see this. Previously the html was commented out to avoid the test failure when we were getting the tests up and running

Reply:
Okay, will have a look.  Commenting this out broke health authority.  Couldn't enter a site name and therefore not submit the page.

Reply:
the test is missing code to initialize the property ` @Input() public doingBusinessAs: FormControl;`
so the following would make it happy
```typescript
  beforeEach(() => {
    fixture = TestBed.createComponent(DoingBusinessAsFormFieldComponent);
    component = fixture.componentInstance;
    component.doingBusinessAs = new FormControl('parent control');
    fixture.detectChanges();
  });
```

Reply:
Thanks Johnny!